### ISSUE TITLE
insert e-mail attachments into SalesLogix tickets

### ISSUE THREAD

**Issue by omsai** [omsai/andorian-hotkeys#1]
Attachments often need to be saved from e-mails into tickets, but this is time consuming.
Below is suggested pseudo code for a new hotkey.
Also, make the key [Outlook specific](http://www.autohotkey.com/docs/FAQ.htm#HotContext) to reduce key pollution.

Sometimes it's necessary to process the attachments (reduce size, discard unnecessary stuff, etc).
In that case it's better to break this into 2 steps-
1.  Create a folder for the ticket and save the file there
2.  _Upload_  folder contents to the ticket

So maybe allow this ability using a GUI with options for:
-  `Save attachments to ticket folder`
-  `Add files to ticket`
-  `Save attachments to ticket via folder`

```
;----------------------------------------------------------------------
; [Windows Key + a] Add attachments to ticket
;----------------------------------------------------------------------
; Inputs:
; - Ticket number copied (i.e. in clipboard)
; - Have e-mail subject highlighted and thus open in Outlook reading pane
;
; Result:
; - Attachment from e-mail added to ticket
; - List of files added to beginning of Description
;----------------------------------------------------------------------
;#a::
; Create desktop folder named with ticket number
; Save attachments in folder
; Import attachments in SLX
; Get attachment names using cmd /K "C: & cd <path\to\folder> & dir /b"
; List attachment names in SLX descripton
;return
```

**Comment by omsai** [comment]
Implemented step 1 of saving attachments in 81722c8e850f62cbd75c55f94cea26a206071f1d

**Comment by omsai** [comment]
This is an unnecessary feature.  Dragging and dropping an e-mail into SLX ticket activities automatically adds the e-mail's attachments.
